src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -492,8 +509,8 @@ export class PatientViewPageStore {
             this.geneticProfilesInStudy
         ],
         invoke: async() => {
-            const profile = this.geneticProfilesInStudy.result.find((profile: GeneticProfile) => {
-                return profile.datatype === 'DISCRETE';
+            const profile = this.geneticProfilesInStudy.result.find((p: GeneticProfile) => {
+                return p.datatype === 'DISCRETE';

```

Comment:
Also, looks like I already moved this function into `StoreUtils` in PR #330. (https://github.com/cBioPortal/cbioportal-frontend/pull/330/files#diff-2a7c36393265d7e597e9d30d3f0314a0R138)

I can handle the conflicts later, it shouldn't be a big deal.